plugins/mcreator-localization/help/fr_FR/block/jump_factor.md

```diff
@@ -0,0 +1,4 @@
+Ce paramètre contrôle la hauteur des sauts des entités sur ce bloc.
+
+La valeur par défault utilisé par la majorité des blocs est 1.

```

Comment:
Should be `La valeur par défaut utilisée par la majorité des blocs est de 1.0.`
You kept the english word for "défaut", you forgot the "é" for "utilisée" and it's more french if you add "de" before 1. Also, in english, you have 1 in double mode, why do you set it in integer mode in French? Keep `1.0`